Branch-SDK/src/io/branch/indexing/BranchUniversalObject.java

```diff
@@ -80,6 +105,8 @@ public BranchUniversalObject() {
         type_ = "";
         indexMode_ = CONTENT_INDEX_MODE.PUBLIC; // Default content indexing mode is public
         expirationInMilliSec_ = 0L;
+        price_ = 0.0;
+        currency_ = CURRENCY_TYPE.NONE;
```

Comment:
in my history i've initialized vars to empty strings to avoid getters throwing nullpointerexceptions.